app/src/main/java/com/vero/tasky/core/domain/util/eventbus/AuthEventBusEvent.kt

```diff
@@ -0,0 +1,7 @@
+package com.vero.tasky.core.domain.util.eventbus
+
+sealed interface AuthEventBusEvent {
+    object UnauthorizedException: AuthEventBusEvent
+    object LogOut: AuthEventBusEvent
+    object LogIn: AuthEventBusEvent
```

Comment:
Why do you need a Login on your autheventbus?

Reply:
I need LogIn to notify my MainViewModel, that current user is loggedIn. If I won't do this and the user will log out at the same session, than login screen will not be changed, because isLoggedIn filed of MainState will be false
![image](https://github.com/AndreVero/Tasky/assets/39765531/359add17-2ba8-49ec-a739-309a45dc4b2e)

Reply:
At the first start this field will be false, and I need to change it after login

Reply:
Mmmm

Cant you check the user is logged in, by checking if the preferences have the token + its authenticated?

Reply:
I still need to notify my MainViewModel about this. Because the user will log-in from the LoginScreen and after that I immediately open AgendaScreen. MainViewModel don't know, that screen was changed after login